supporting_content/dimensions/skupriceid.md

```diff
@@ -0,0 +1,39 @@
+## Example provider mappings 
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                | Column                             |
+| ------------ | --------------------------- | -------------------------------------- |
+| AWS          | CUR                         | pricing/rate\_code \| pricing/rate\_id |
+| Azure        | Cost details export or API  | ProductID                              |
+| GCP          | BigQuery Billing Export            | Not available, but can be derived from sku.id and price.tier_start_amount                                 |
+| OCI          | Cost Reports                | Not available    |
```

Comment:
```suggestion
| OCI          | Cost Reports                | Not available, no price level ID    |
```

Reply:
Adding a justification for "Not available" so future reviewers know what is required for a mapping to be valid.